.claude/skills/pr-review-todoapp/SKILL.md

```
@@ -0,0 +1,417 @@
+
+**インストール確認:**
+
+```bash
+gh pr-review --help
+```
+
+---
+
+## Workflow Overview
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)